builder/Dockerfile

```diff
@@ -33,31 +26,42 @@ RUN dnf -y update \
         libtool \
         libuuid-devel \
         make \
-        openssh-server \
         openssl-devel \
-        patchutils \
-        passwd \
         pkgconfig \
-        python2 \
-        rsync \
-        tar \
-        unzip \
         valgrind \
         wget \
-        which \
     && dnf clean all
 
 # Build dependencies from source
 WORKDIR ${BUILD_DIR}
 
-COPY builder builder
+COPY install builder/install
+COPY build builder/build
 COPY third_party third_party
 
-RUN "builder/install/install-dependencies.sh"
+RUN "builder/install/install-dependencies.sh" && \
+    rm -rf ${BUILD_DIR} && \
+    echo '/usr/local/lib' > /etc/ld.so.conf.d/usrlocallib.conf && ldconfig
 
-RUN rm -rf ${BUILD_DIR}
+FROM builder as dev
 
-RUN echo '/usr/local/lib' > /etc/ld.so.conf.d/usrlocallib.conf && ldconfig
+RUN dnf -y update \
+    && dnf -y install --nobest \
+        bison \
+        ca-certificates \
+        clang \
+        cracklib-dicts \
```

Comment:
cracklib-dicts was a dependency needed for running sshd -- but that is an outdated development model. I currently use the [docker toolchain](https://www.jetbrains.com/help/clion/clion-toolchains-in-docker.html) in Clion which no longer needs sshd. If sshd isn't needed by any other development workflows I suggest that we remove it.

Reply:
@JoukoVirtanen I believe you are also using CLion, could you confirm you also don't need `sshd` anymore so I can remove it?

Reply:
Just chatted with Jouko and he is no longer using this, so I'll go ahead and drop the entire ssh workflow